observatorium/overlays/moc/smaug/thanos/obcs/observatorium-bucket.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: objectbucket.io/v1alpha1
+kind: ObjectBucketClaim
+metadata:
+  name: observatorium-obc
```

Comment:
we need 2 different OBCs
1 for opf-observatorium-logs
1 for opf-observatorium-metrics